mon-pix/app/components/account-recovery/backup-email-confirmation-form.js

```diff
@@ -27,9 +27,14 @@ export default class BackupEmailConfirmationFormComponent extends Component {
 
   @service intl;
 
+  @tracked email = '';
   @tracked emailValidation = new EmailValidation();
 
-  email = '';
+  get isSubmitButtonEnabled() {
+    return isEmailValid(this.email)
+      && this.emailValidation.status !== STATUS_MAP['errorStatus']
+      && !this.args.isLoading;
+  }
```

Comment:
**question**
Si `this.emailValidation.status !== STATUS_MAP['errorStatus']` fait référence à l'API

```suggestion
  get isSubmitButtonEnabled() {
    return isEmailValid(this.email)
      && !this._hasAPIRejectedCall()
      && !this.args.isLoading;
  }

  _hasAPIRejectedCall() {
    return this.emailValidation.status === STATUS_MAP['errorStatus'];
  }

```

Reply:
Done